mobile/lib/screens/feedback/feedback_page_widgets.dart

```diff
@@ -597,7 +597,7 @@ class _FeedbackChannelStepState extends State<FeedbackChannelStep> {
           height: 32,
         ),
         Text(
-          'Send Us Feedback Via Email Or Whatsapp',
+          'Send Us Feedback Via Email',
```

Comment:
I hope the email option actually works?
Also, how long could this update take? Thanks again @NoahNsimbe